.github/workflows/dotnet.yaml

```diff
@@ -38,7 +38,7 @@ jobs:
     - name: Build
       run: 'nix develop --command dotnet build --no-restore --configuration ${{matrix.config}}'
     - name: Test
-      run: 'nix develop --command dotnet test --no-build --verbosity normal --configuration ${{matrix.config}}'
+      run: 'nix develop --command dotnet test --no-build --verbosity normal --configuration ${{matrix.config}} --framework net8.0'
```

Comment:
Bug in .NET, I think: runtime 6 is on the path, but `dotnet` isn't finding it, presumably because after symlink resolution the `dotnet` in question is isolated inside a net8 SDK.